package.json

```
@@ -9,6 +9,29 @@
   "files": [
     "**"
   ],
+  "exports": {
+    "./package.json": "./package.json",
+    ".": {
+      "types": "./index.d.ts",
+      "import": "./index.js",
```

```suggestion
      "module": "./index.js",
```
react doesn't support mjs, so we should only enable "import" for pure libs. let's make it back "module" for webpack5.

---

hm, I was confused with something else (= type="module"). Should be fine.